test/system/current_user/reports_test.rb

```diff
@@ -7,4 +7,16 @@ class CurrentUser::ReportsTest < ApplicationSystemTestCase
     visit_with_auth '/current_user/reports', 'hatsuno'
     assert_equal '自分の日報 | FJORD BOOT CAMP（フィヨルドブートキャンプ）', title
   end
+
+  test 'show reports download btn when reports exist' do
```

Comment:
```suggestion
  test 'show reports download button when reports exist' do
```

特に理由がなければ略さない方がいいかも？

Reply:
@komagata
特に理由なく略していました... 気をつけます🙇‍♂️
修正したものをプッシュしたので、再度確認よろしくお願いします！